.github/ISSUE_TEMPLATE/release.md

```diff
@@ -31,6 +31,8 @@ Replace the items in the task lists below with the applicable Pull Requests / Is
 
 ```[tasklist]
 ### Early Pre-release tasks
+- [ ] Create a "Post XX.XX release" retro issue (does this need a template? See https://github.com/stackabletech/issues/issues/669)
+- [ ] Define product versions to include in the next release
```

Comment:
> note: We should mention that it might be clever to already list release note worthy product features during this process.

We could, but it makes this ticket rather busy. It is already covered in the product update template: https://github.com/stackabletech/docker-images/blob/d27aec50f27b0b19a9678612099c3eb9658c6268/.github/ISSUE_TEMPLATE/update-product-airflow.md?plain=1#L57-L58

Reply:
Ah yeah, perfect!